src/templates/hooks/post-deploy.js

```diff
@@ -1,8 +1,20 @@
 const chalk = require('chalk')
+const fs = require('fs')
+const yaml = require('js-yaml')
 
 module.exports = (config) => {
     console.log(chalk.magenta(chalk.bold('For a developer preview of your UI extension in the AEM environment, follow the URL:')))
-    const appUrl =  `https://${config.ow.namespace}.${config.app.hostname}`
-    const base64EncodedUrl = Buffer.from(appUrl).toString('base64')
-    console.log(chalk.magenta(chalk.bold(`  -> https://experience.adobe.com/aem/extension-manager/preview/${base64EncodedUrl}`)))
+    const yamlFile = fs.readFileSync(`${config.root}/app.config.yaml`, 'utf8')
+    const yamlData = yaml.load(yamlFile)
+    const { extensions } = yamlData
+    const appUrl = `https://${config.ow.namespace}.${config.app.hostname}`
+
+    // For now we are ok just to read the first extension point to build the preview link
+    const extension = Object.keys(extensions)[0]
```

Comment:
Have you tested that this code works as expected I'm the following use cases?
- deploy an app with no extension points: 
- Deploy the app with extension points not supported by the extension manager.

Reply:
Since this template was specifically designed for the CF editor extension point, I'd prefer not to overcomplicate it with additional conditional logic.
However, I did wrap everything with a try-catch so that if something goes wrong, we will not display the message from this hook.